src/contexts/NotificantionManager/NotificationManagerProvider.tsx

```diff
@@ -0,0 +1,21 @@
+import { ReactNode, useMemo } from 'react';
+import useNotifications from './useNotifications';
+import { NotificationManagerContext } from './NotificationManagerContext';
+import NotificationComponent from './NotificationComponent';
+
+// const Notificaton = dynamic(() => import('./NotificationComponent'), { ssr: false });
```

Comment:
to remove?

Reply:
done 573926c86b8b1e62dade77fa67e287d954d8d93d